src/App.tsx

```diff
@@ -1,18 +1,30 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.css';
 import { getNumbers } from './utils';
+import { Pagination } from './components/Pagination';
 
-// eslint-disable-next-line @typescript-eslint/no-unused-vars
 const items = getNumbers(1, 42)
   .map(n => `Item ${n}`);
 
 export const App: React.FC = () => {
+  const [currentPage, setCurrentPage] = useState(1);
+  const [perPage, setPerPage] = useState(5);
+
+  const visibleItems
+  = items.filter((item, index) => item
```

Comment:
```suggestion
  const visibleItems = items.filter((item, index) => item
```
![image](https://github.com/mate-academy/react_pagination/assets/96409563/442d9b7a-4eb6-4539-8f08-bbc3adda88f0)